ui/lib/pki/addon/components/pki-role-form.hbs

```diff
@@ -2,35 +2,77 @@
   <div class="box is-sideless is-fullwidth is-marginless">
     <MessageError @errorMessage={{this.errorBanner}} class="has-top-margin-s" />
     {{! ARG TODO write a test for namespace reminder }}
-    <NamespaceReminder @mode={{if @model.isNew "create" "update"}} @noun="PKI role" />
-    {{#each @model.formFieldGroups as |fieldGroup|}}
+    <NamespaceReminder @mode={{if @role.isNew "create" "update"}} @noun="PKI role" />
+    {{#each @role.formFieldGroups as |fieldGroup|}}
       {{#each-in fieldGroup as |group fields|}}
         {{! DEFAULT VIEW }}
         {{#if (eq group "default")}}
           {{#each fields as |attr|}}
-            <FormField
-              data-test-field={{attr}}
-              @attr={{attr}}
-              @model={{@model}}
-              @modelValidations={{this.modelValidations}}
-              @showHelpText={{false}}
-            >
-              <PkiNotValidAfterForm @attr={{attr}} @model={{@model}} />
-            </FormField>
+            {{#if (eq attr.name "issuerRef")}}
+              <div class="has-top-margin-m {{unless this.showDefaultIssuer 'has-bottom-margin-xs' 'has-bottom-margin-m'}}">
+                <FormFieldLabel
+                  for={{attr.name}}
+                  @label="Issuer ref"
+                  @helpText={{(if this.showHelpText attr.options.helpText)}}
+                  @subText={{attr.options.subText}}
+                />
+                <Toggle
+                  @name={{concat attr.name "-toggle"}}
+                  @status="success"
+                  @size="small"
+                  @checked={{this.showDefaultIssuer}}
+                  @onChange={{this.toggleShowDefaultIssuer}}
+                >
+                  <span class="has-text-grey">Use default issuer</span>
+                </Toggle>
+              </div>
+              {{#unless this.showDefaultIssuer}}
+                <div class="has-top-margin-xs has-bottom-margin-l">
+                  <div class="select is-fullwidth">
```

Comment:
A discovery I had through the bulma work, we do this crazy thing on the select component: we have a div with a class name="select" wrap a select element with a class name="select". Literally this looks like:
`.select 
   select.select {
// styling
  }

Anyways, I point that out because we do have a Select component (it has that syntax). It's old. But I want to make sure that component couldn't work for our purposes here?

Reply:
Thanks for pointing that out! I didn't realize we had a `Select` component. Looking at the`Select` component, it looks like should work here as well. I will update!